helm/nessie/templates/servicemonitor.yaml

```diff
@@ -21,15 +21,14 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ template "nessie.fullname" . }}
-  namespace: {{ .Release.Namespace }}
```

Comment:
Unrelated, but see #9620.

Reply:
So this is rather the "opposite" of #9621, but addresses the same issue, correct?

Reply:
Yes, I wanted to make things consistent, even if I actually agree with the fix proposed in #9621.